app/assets/stylesheets/static_page.scss

```diff
@@ -0,0 +1,12 @@
+.static-page {
+    margin: 0 auto;
+    padding-top: 1.8rem;
```

Comment:
on essaie au max de se passer de css custom. Par exemple pour tout ce qui est espacement, on a des helpers du dsfr type `fr-pt-2w` ou `fr-my-2w` disponibles.
https://www.systeme-de-design.gouv.fr/elements-d-interface/fondamentaux-techniques/espacement

Et pour la structure de la page (qui nécessite ici le margin auto) il y a le système e grille qui fait ça, on a commencé à l'utiliser sur certaines pages
https://www.systeme-de-design.gouv.fr/elements-d-interface/fondamentaux-techniques/grille-et-points-de-rupture

Reply:
C'est noté Colin :)